config/locales/en.yml

```diff
@@ -462,12 +462,12 @@ en:
       submit_button: Submit button
     new_analysis_export_dialog:
       description:
-        plural: This export will contain COUNT_PLACEHOLDER workflow executions.
-        singular: This export will contain 1 workflow execution.
-        zero: This export will contain no workflow executions.
+        plural: This export will contain COUNT_PLACEHOLDER pipeline executions.
```

Comment:
**nitpick (blocking):** Can you revert the following 7 translation changes

Reply:
Updated in 51f582d